testbench.yaml

```diff
@@ -1,6 +1,5 @@
 providers:
   - Laravel\Nightwatch\NightwatchServiceProvider
-  - App\Providers\AppServiceProvider
```

Comment:
Migrated this to the conventional `workbench/bootstrap/providers.php` file while installing the local Horizon service provider, which is not auto-detected.